src/components/calcite-slider/calcite-slider.e2e.ts

```diff
@@ -196,6 +196,17 @@ describe("calcite-slider", () => {
     expect(changeEvent).toHaveReceivedEventTimes(1);
   });
 
+  it("maxValue label in range slider should have tick__label--max class", async () => {
```

Comment:
The missing class affected the behavior of overlapping labels, right? Could we add stories to cover this instead of asserting on the class being there? 

This commit could be helpful as it points to a version of the demo page with configurations for all of the overlapping scenarios: https://github.com/Esri/calcite-components/blob/cd16695ee0546b4d89f220d3f5f0aabee4e6c06b/src/demos/calcite-slider.html#L246